docs/generate-tables.sh

```diff
@@ -1,31 +1,30 @@
 #!/bin/bash
 # Generate the documentation tables as docs/generated/OJP.html from the .xsd schema files
 #
-# You need the binary `xsltproc`
+# You need the binary `java`
 # apt-get install xsltproc
 
 # The -e flag causes the script to exit as soon as one command returns a non-zero exit code
 set -e
 
-echo "Generating documentation tables ..."
-
-base_dir="$(dirname "${0}")/.."
+base_dir="$(readlink -f $(dirname "$0")/..)"
 xsl_dir=$base_dir/docs
 generated_dir="${base_dir}/docs/generated"
+basex="/tmp/basex.jar"
 
-# prepare generated_dir
-mkdir -p "${generated_dir}"
-rm -f "${generated_dir}"/OJP-prep.xml "${generated_dir}"/*.adoc "${generated_dir}"/*.html
-cp "${xsl_dir}"/asciidoc.css "${generated_dir}"/
+echo "Download BaseX ..."
 
-# create intermediate XML file for documentation
-xsltproc --xinclude "${xsl_dir}"/ojp-to-prepdoc.xsl \
- "${xsl_dir}"/schema-collection.xml \
- >> "${generated_dir}"/OJP-prep.xml
+if [ ! -e ${basex} ]; then
+	wget --output-document=${basex} https://files.basex.org/releases/10.6/BaseX106.jar
+fi
 
-# generate stand-alone HTML file for documentation
-xsltproc --xinclude "${xsl_dir}"/ojp-prep-to-html-with-toc.xsl \
- "${generated_dir}"/OJP-prep.xml \
- >> "${generated_dir}"/index.html
+echo "Generating documentation tables ..."
+
+# prepare generated_dir
+mkdir -p "${generated_dir}"
+rm -f "${generated_dir}"/contab/*.html
```

Comment:
What does "contab" mean and why are the files in this subdirectory instead of directly in generated/* ?

Reply:
I have absolutely no idea.

Reply:
Here what I found about the term "contab" (from the documentation):

![grafik](https://github.com/VDVde/OJP/assets/82838870/83e0de42-f790-4766-bd34-92b6fcd1d666)